### ISSUE TITLE
The insiders version 2.7.0 breaks git-revision-date-localized plugin

### ISSUE THREAD

**Issue by hrobarikt** [squidfunk/mkdocs-material#2630]
- [x] I've read the [contribution guidelines][1] and agree with them

__I've found a bug and checked that ...__

- [x] ... the problem doesn't occur with the default MkDocs template
- [x] ... the problem is not in any of my customizations (CSS, JS, template)
- [x] ... the documentation does not mention anything about my problem
- [x] ... there are no open or closed issues that are related to my problem

## Description
When using the latest version of insiders 2.7.0 which support tags, I'm unable to use [git-revision-date-localized plugin](https://squidfunk.github.io/mkdocs-material/setup/adding-a-git-repository/#revision-date-localized). The build will fail with the following error:
```
ERROR   -  Error building page 'index.md': source-file.html
Traceback (most recent call last):
  File "/opt/homebrew/bin/mkdocs", line 8, in <module>
    sys.exit(cli())
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1259, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/mkdocs/__main__.py", line 152, in build_command
    build.build(config.load_config(**kwargs), dirty=not clean)
  File "/opt/homebrew/lib/python3.9/site-packages/mkdocs/commands/build.py", line 292, in build
    _build_page(file.page, config, files, nav, env, dirty)
  File "/opt/homebrew/lib/python3.9/site-packages/mkdocs/commands/build.py", line 210, in _build_page
    output = template.render(context)
  File "/opt/homebrew/lib/python3.9/site-packages/jinja2/environment.py", line 1090, in render
    self.environment.handle_exception()
  File "/opt/homebrew/lib/python3.9/site-packages/jinja2/environment.py", line 832, in handle_exception
    reraise(*rewrite_traceback_stack(source=source))
  File "/opt/homebrew/lib/python3.9/site-packages/jinja2/_compat.py", line 28, in reraise
    raise value.with_traceback(tb)
  File "/opt/homebrew/lib/python3.9/site-packages/material/main.html", line 4, in top-level template code
    {% extends "base.html" %}
  File "/opt/homebrew/lib/python3.9/site-packages/material/base.html", line 178, in top-level template code
    {% block content %}
  File "/opt/homebrew/lib/python3.9/site-packages/material/base.html", line 192, in block "content"
    {% include "partials/source-date.html" %}
  File "/opt/homebrew/lib/python3.9/site-packages/material/partials/source-date.html", line 4, in top-level template code
    {% include "source-file.html" %}
  File "/opt/homebrew/lib/python3.9/site-packages/jinja2/loaders.py", line 197, in get_source
    raise TemplateNotFound(template)
jinja2.exceptions.TemplateNotFound: source-file.html
```

### Expected behavior
Ability to use the git-revision-date-localized plugin together with tags support.

### Actual behavior
Unable to build mkdocs site with latest insiders version together with git-revision-date-localized plugin

### Steps to reproduce the bug

1. install the latest insiders version 2.7.0 
2. setup the blank project in your folder `mkdocs new .`
3. install the plugin `pip install mkdocs-git-revision-date-localized-plugin`
4. modify `mkdocs.yml` as described below
5. run `mkdocs build`

### Package versions

* Python: `python --version` 2.7.16
* MkDocs: `mkdocs --version` version 1.1.2 from /opt/homebrew/lib/python3.9/site-packages/mkdocs (Python 3.9)
* Material: `pip show mkdocs-material | grep -E ^Version` Version: 7.1.3+insiders.2.7.0

### Project configuration

``` yaml
site_name: My Docs
theme:
  name: material
plugins:
  - git-revision-date-localized:
      fallback_to_build_date: true
```

### System information

* OS: MacOS Big Sur 11.3
* Browser: MS Edge

  [1]: https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md

**Comment by squidfunk** [comment]
Thanks for reporting! It's not specific to Insiders but Material for MkDocs in general.

Related issues / PRs:

- #2627
- #2629
- #2579

**Comment by squidfunk** [comment]
Fixed and released as part of Insiders 2.7.1.
